src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -288,7 +294,16 @@ Run the `pnpm db:init` script as it will populate the database with an initial s
 `db:seed` is responsible to populate the database with an initial set of data.
 :::
 
-Stop the `parser` service running. If you have Docker app installed locally make sure to stop the `parser` container before continuing to the next step.
+Stop the `parser` service running. Make sure to stop the `parser` container before continuing to the next step by either using
+
+Docker Desktop app:
+![Original Telescope architecture](../static/img/initialProjectDiagram.png)
+
+Docker CLI: `docker stop <container-id>` (e.g, `15649f07c6bf` in above picture)
+
+To find container-id: `docker container ls -a` This will return all the docker containers, STATUS, and their PORTS
+
+Switch to app's `.env` by `cp ./config/env.development ./.env` (you changed `.env` previously to seed database).
```

Comment:
We can remove this line, @manekenpix explained it well, development's `.env` is already built in docker containers